components/shared-assets/illustrations/cloud.tsx

```diff
@@ -107,8 +107,8 @@ export const md = ({
     ...otherProps
 }: Omit<IllustrationProps, "size">) => {
     return (
-        <div {...otherProps} className={cx("relative", className)}>
-            <svg width="172" height="128" viewBox="0 0 182 137" fill="none" className={cx("size-full stroke-inherit text-inherit", svgClassName)}>
+        <div {...otherProps} className={cx("relative h-[128px] w-[172px]", className)}>
```

Comment:
Here you could use h-32 w-44

Reply:
@franklinjavier Sorry I just saw this. Created a new PR to tackle these https://github.com/untitleduico/react/pull/35. Thanks for the suggestion!